src/Podium/Request.ts

```diff
@@ -1,16 +1,47 @@
-import axios, {AxiosError, AxiosRequestConfig} from 'axios'
+import axios, {AxiosError, AxiosRequestConfig, AxiosResponse} from 'axios'
 import {API_CODE, IPodiumErrorResponse, IPodiumPromise, IResponse, ISettings} from '../../types'
 import {ConvertTime} from './ConvertTime'
 import {Filter} from './Filter'
 import {Token} from './Token'
 import {Paginator} from './Paginator'
+import {Settings} from './Settings'
 
 export class Request extends Token {
+    private static parseError(error: AxiosError): IPodiumErrorResponse {
+        if (error.response) {
+            const message =
+                (typeof error.response.data === 'object' &&
+                    (error.response.data.message
+                        || (error.response.data.detail && Object.values(error.response.data.detail)
+                            .map((errorDetail)  =>
+                                (typeof errorDetail === 'string' && errorDetail)))
+                        || Object.values(error.response.data)[0])
+                ) || error.response
+            return {
+                code: error.response.data.code,
+                data: error.response.data as IResponse,
+                message,
+                status: error.response.status,
+                statusText: error.response.statusText,
+            }
+        } else {
+            // todo: fix this
```

Comment:
What is wrong with this?

Reply:
So AxiosError might not have .response, so i had to add if and else part i came up with randomly

Reply:
Can you  at lease add that to the comment;